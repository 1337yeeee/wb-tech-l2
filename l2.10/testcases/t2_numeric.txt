10
2
33
1
