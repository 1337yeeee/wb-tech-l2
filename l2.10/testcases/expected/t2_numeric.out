1
2
10
33
